daemon-tests/tests/happy_path.rs

```diff
@@ -427,6 +427,42 @@ async fn maker_rejects_rollover_after_commit_finality() {
     wait_next_state!(order_id, maker, taker, CfdState::OpenCommitted);
 }
 
+#[tokio::test]
+async fn maker_accepts_rollover_after_commit_finality() {
```

Comment:
```suggestion
async fn given_maker_committed_when_accept_rollover_then_rolls_over() {
```

I think it should actually be:

```suggestion
async fn given_maker_committed_when_accept_rollover_then_remains_comitted() {
```

and fail...

Reply:
happy to have a further chat about these - the test names you proposed are IMHO less legible (at least for me)